packages/extension-base/ctp-extension-azure-adapter.js

```diff
@@ -0,0 +1,32 @@
+// Azure
+
+const createReject = ctx => error => {
+  ctx.res = {
+    status: 400,
+    body: {
+      errors: Array.isArray(error) ? error : [error],
+    },
+  };
+  ctx.done();
+};
+
+const createAccept = ctx => (update = []) => {
+  ctx.res = {
+    status: 200,
+    body: {
+      actions: Array.isArray(update) ? update : [update],
+    },
+  };
+  ctx.done();
+};
+
+const ctpResponse = ctx => ({
+  reject: createReject(ctx),
+  accept: createAccept(ctx),
+});
+
+module.exports = {
+  create: fn => (ctx, req) => {
```

Comment:
Slight API change for a `adapter.create(fn)(ctx, res)`

Reply:
I do expect to *eventually* create an adapter for subscriptions as well.

I'm fine with using `create` in the function name, but I'm wondering if it isn't better to be more explicit (e.g. `createExtensionAdapter` and `createSubscriptionAdapter`).

Reply:
Ah yeah sure. Whatever you feel is more sustainable.